documentation/src/main/asciidoc/upgrading/upgrading-content.adoc

```diff
@@ -32,6 +32,11 @@ With the introduction of inline XML JGroups stacks in the configuration, two def
 If you are declaring your own stacks with the same names, an exception reporting the conflict will be thrown. Simply rename
 your own configurations to avoid the conflict.
 
+=== Cache and Cache Manager Listeners can now be configured to be non blocking
+Listeners in the past that were sync, always ran in the thread that caused the event. We now allow a Listener method to
+be non-blocking in that it will still fire in the original thread, under the assumption that it will return immediately.
```

Comment:
This needs updating

Reply:
It was updated.. What were you thinking?